frontend/src/layout/Sidebar.js

```diff
@@ -54,51 +59,91 @@ export default function Sidebar(props) {
     const { scene, loadingScene } = useValues(sceneLogic)
     const { location } = useValues(router)
     const { push } = useActions(router)
+    const { dashboards, pinnedDashboards } = useValues(dashboardsModel)
 
-    const activeScene = sceneOverride[loadingScene || scene] || loadingScene || scene
+    let activeScene = sceneOverride[loadingScene || scene] || loadingScene || scene
+    const openSubmenu = submenuOverride[activeScene] || activeScene
+
+    let firstDashboard = pinnedDashboards.length > 0 ? `/dashboard/${pinnedDashboards[0].id}` : '/dashboard'
+    let unpinnedDashboard = null
+    if (activeScene === 'dashboard') {
+        const dashboardId = parseInt(location.pathname.split('/dashboard/')[1])
+        activeScene = `dashboard-${dashboardId}`
+
+        const dashboard = dashboards.find(d => d.id === dashboardId)
+        if (dashboard && !dashboard.pinned) {
+            unpinnedDashboard = dashboard
+        }
+    }
 
     return (
-        <Layout.Sider breakpoint="lg" collapsedWidth="0" className="bg-light">
+        <Layout.Sider breakpoint="lg" collapsedWidth="0" className="bg-dark">
             <Menu
-                className="h-100 bg-light"
+                className="h-100 bg-dark"
+                theme="dark"
                 selectedKeys={[activeScene]}
-                openKeys={[submenuOverride[activeScene] || activeScene]}
+                openKeys={[openSubmenu]}
                 mode="inline"
             >
                 <Logo />
                 <Menu.Item key="trends" style={itemStyle}>
                     <RiseOutlined />
-                    <span>{'Trends'}</span>
+                    <span className="sidebar-label">{'Trends'}</span>
                     <Link to={'/trends'} />
                 </Menu.Item>
-                <Menu.Item key="dashboard" style={itemStyle}>
-                    <HomeOutlined />
-                    <span>{'Dashboard'}</span>
-                    <Link to={'/dashboard'} />
-                </Menu.Item>
+                <Menu.SubMenu
+                    key="dashboards"
+                    title={
+                        <span style={itemStyle}>
+                            <HomeOutlined />
+                            <span className="sidebar-label">Dashboards</span>
+                        </span>
+                    }
+                    onTitleClick={() => (location.pathname !== firstDashboard ? push(firstDashboard) : null)}
```

Comment:
Combined with my comment below, when clicking on this I expect to go to 'all dashboards'